CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 🍹 [4.8.1](https://github.com/ionic-team/stencil/compare/v4.8.0...v4.8.1) (2023-12-04)
+
+
+### Bug Fixes
+
+* **compiler:** ensure not to import duplicate style identifier ([#5119](https://github.com/ionic-team/stencil/issues/5119)) ([7591dce](https://github.com/ionic-team/stencil/commit/7591dce85e3b19ab623752341344d009881032be))
+* **compiler:** revert changes to style import transformer ([#5125](https://github.com/ionic-team/stencil/issues/5125)) ([cf4a701](https://github.com/ionic-team/stencil/commit/cf4a70150c91e9dcc6690396a7e1b3a0a1e92215))
+* **compiler:** support multiple styleUrls in components ([#5090](https://github.com/ionic-team/stencil/issues/5090)) ([54e52da](https://github.com/ionic-team/stencil/commit/54e52da952faa59026c00cf603a027dae5fe82ee))
```

Comment:
I _think_ we'll want to remove these, since the second entry reverts the other two